ssdutil/main.py

```diff
@@ -16,11 +17,29 @@
 
 DEFAULT_DEVICE="/dev/sda"
 SYSLOG_IDENTIFIER = "ssdutil"
+DISK_TYPE_SSD = "0"
 
 # Global logger instance
 log = logger.Logger(SYSLOG_IDENTIFIER)
 
 
+def get_disk_type(diskdev):
+    """Check disk type"""
+    diskdev_name = diskdev.replace('/dev/','')
+    cmd = "lsblk -l -n |grep disk"
+    proc = subprocess.Popen(cmd, shell=True, text=True, stdout=subprocess.PIPE)
+    outs = proc.stdout.readlines()
+    for out in outs:
+        if out.split()[0] is diskdev_name:
```

Comment:
what if diskdev_name is one of  ['sda', 'sdb', 'sdc'], but you seem to be checking only the first element